src/backend/sev/snp/ghcb.rs

```diff
@@ -0,0 +1,57 @@
+// SPDX-License-Identifier: Apache-2.0
+
+// TODO: Consider sharing these types with the shim.
```

Comment:
This pr would benefit from #2451. It'd probably be better to just share the types in this file in a common crate.